### PR TITLE
[Merge] Main -> Development

### PR DESCRIPTION
**Pull request by LefaJele-Masemola**
Merging main -> dev to get dev up to date with main

### CODE REVIEW

**Review by Lord-Nickelbottoms — APPROVED** [review]

**Comment by SilindileShabangu** [comment]
Perfect, but please change the link for the button from Blank

File: portfolio-landing-page/chatbot/script.js [hunk 1]
```diff
@@ -66,13 +78,13 @@ function generateCreativeBotReply(userMessage) {
     if (message.includes("hello") || message.includes("hi")) {
         return "Hey there! 👋 What's up?";
     } else if (message.includes("portfolio") || message.includes("projects")) {
-        return "I'm glad you're interested! Check out my favorite projects, <a href=\"index.html#projects\" target=\"_blank\">here</a>, and feel free to ask about any of them!";
```
Comment (SilindileShabangu) [inline portfolio-landing-page/chatbot/script.js:69]: remove blank here

**Review by Ayanda-Radebe — APPROVED** [review]
it looks good!

**Review by Lord-Nickelbottoms — APPROVED** [review]
Looking clean :). Everything looks good.

**Review by SilindileShabangu — DISMISSED** [review]

**Review by Sisamkele022 — APPROVED** [review]
code readability looks good

**Review by Sisamkele022 — APPROVED** [review]

**Review by SilindileShabangu — APPROVED** [review]

**Pull request merged**